astrokat/test/astrokat_testutils.py

```diff
@@ -0,0 +1,42 @@
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2019 SKA SA. All rights reserved.                               #
+#                                                                             #
+# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
+# WRITTEN PERMISSION OF SKA SA.                                               #
```

Comment:
This is the incorrect license for astrokat.  It is released under BSD-2.  Looking at the other files, we aren't putting copyright blocks in every file - just relying on the LICENSE file at the top level.